Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -2593,6 +2594,96 @@ public void Editor_CanRestartEditorThroughReflection()
         EditorHelpers.RestartEditorAndRecompileScripts(dryRun: true);
     }
 
+    [Test]
+    public void DocsContainNoMissingOrUnreferencedImages()
```

Comment:
Just to keep them together, would prefer to have them in the `APIVerificationTests` suite alongside `API_DocumentationManualDoesNotHaveMissingInternalLinks` test. Neither is quite about API verification but at least they'll be in the same place.

```
    [Test]
    [Category("API")]
    public void API_DocumentationManualDoesNotContainMissingOrUnreferencedImages()
```